src/Compilers/Core/MSBuildTaskTests/CscTests.cs

```diff
@@ -286,5 +286,19 @@ public void Embed()
             csc.EmbeddedFiles = MSBuildUtil.CreateTaskItems();
             Assert.Equal(@"/debug:portable /out:test.exe test.cs", csc.GenerateResponseFileContents());
         }
+
+        [Fact]
+        public void EditorConfig()
```

Comment:
Added tests for the first two .The third was already covered in the second scenario inside this test. (But my new one covers that too.)